filebeat/module/shorewall/log/ingest/pipeline.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "Pipeline for parsing Shorewall firewall. Requires the geoip plugin.",
+    "processors": [
+        {
+            "grok": {
+                "field": "message",
+                "patterns": [
+                    "(%{SYSLOGTIMESTAMP:timestamp}) (%{WORD:host.hostname}) kernel:.*Shorewall:(%{WORD:shorewall.action.one})?:(%{WORD:shorewall.action.two})?.*IN=(%{USERNAME:shorewall.network.in})?.*(OUT= *MAC=(%{COMMONMAC:destination.mac}):(%{COMMONMAC:source.mac})?|OUT=%{USERNAME:shorewall.network.out}).*SRC=(%{IPV4:source.ip}).*DST=(%{IPV4:destination.ip}).*LEN=(%{INT:network.packets:int}).?*TOS=(%{WORD:service.type}).?*PREC=(%{WORD:shorewall.precedence}).?*TTL=(%{INT:shorewall.ttl:int}).?*ID=(%{INT:event.id:int}).?*PROTO=(%{WORD:network.protocol}).?*SPT=(%{INT:source.port:int}?.*DPT=%{INT:destination.port:int}?.*)"
+                ],
+                "ignore_missing": false
+            }
+        },
+        {
+            "date": {
+                "field": "timestamp",
```

Comment:
Timezone is missing. Here you need the convert timezone plugin. Check another module for guidance about how to use it (or this PR https://github.com/elastic/beats/pull/12079), it will make the document non JSON valid but **this is expected** when using the convert timezone plugin